Procfile.dev.example

```diff
@@ -1,2 +1 @@
-pg: postgres -D tmp/postgres -p 5432
-web: FLASK_DEBUG=1 flask run
```

Comment:
here I'd add the `python manage.py runserver` command as the web part, so that one can quickly get both Django and postgres up and running using `honcho`.